pkg/volume/azure_dd/mounter.go

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
the file is named azure_dd.go, as for mount and attacher i didn't find a consistent naming concision across the plugins. I can prefix all files with azure_dd if that will help.